pkg/sentry/kernel/msgqueue/msgqueue.go

```diff
@@ -129,6 +129,36 @@ type Message struct {
 	mSize uint64
 }
 
+// NewMessage returns a new, initialized Message.
+func NewMessage(mType int64, mText []byte, mSize uint64) Message {
```

Comment:
If the fields are public, you can also drop this constructor, and the caller can create the struct directly.

Reply:
Done